libs/langchain/langchain/storage/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class LangChainException(Exception):
```

Comment:
feels like this should be somewhere more general

Reply:
like schema

Reply:
I'll move to `schema.exceptions` for LangChainException, and keep the InvalidKeyException here for the time being